Cargo.toml

```diff
@@ -18,56 +18,56 @@ include = ["../../LICENSE-APACHE", "../../LICENSE-MIT", "**/*.rs", "Cargo.toml"]
 license = "MIT OR Apache-2.0"
 repository = "https://github.com/rerun-io/rerun"
 rust-version = "1.72"
-version = "0.10.0"
+version = "0.11.0-alpha.1+dev"
```

Comment:
Can you remind me what the +dev does here?

Reply:
It means "not-yet-released" - it is not the published `alpha.1`, but it is building towards it

Reply:
That means it's an unreleased version, so it's clearly marked as such in the various builds we do both locally and on CI.

The versioning scheme is supposed to be `0.10.0` -> `0.11.0-alpha.1+dev` -> `0.11.0-alpha.1` -> `0.11.0-alpha.2+dev` -> `0.11.0-rc.1` -> `0.11.0` -> `0.12.0-alpha.1+dev` -> etc.

Reply:
Makes sense! Thanks.